borg/archiver.py

```diff
@@ -1140,15 +1093,22 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('--short', dest='short',
                                action='store_true', default=False,
                                help='only print file/directory names, nothing else')
-        subparser.add_argument('--list-format', dest='listformat', type=str,
-                               help="""specify format for archive file listing
-                                (default: "{mode} {user:6} {group:6} {size:8d} {isomtime} {path}{extra}{NEWLINE}")
-                                Special "{formatkeys}" exists to list available keys""")
+        subparser.add_argument('-f', '--format', '--list-format', dest='format', type=str,
```

Comment:
Agreed. --format is short enough anyway, --list-format is kinda "strange" however (*what other format would it be?*)